include/zephyr/dt-bindings/regulator/cp9314.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Cirrus Logic, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_REGULATOR_CP9314_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_REGULATOR_CP9314_H_
+
+/**
+ * @defgroup regulator_cp9314 CP9314 Devicetree helpers.
+ * @ingroup regulator_interface
+ * @{
+ */
+
+/**
+ * @name CP9314 Regulator modes
+ * @{
+ */
+/** 2:1 mode */
+#define CP9314_MODE_2TO1 1
+/** 3:1 mode */
+#define CP9314_MODE_3TO1 2
+/** @} */
```

Comment:
can datasheet or details about these modes be provided? are these regulation modes?

Reply:
ack https://github.com/zephyrproject-rtos/zephyr/pull/68177#issuecomment-1917431116